pkg/syncer/metadatasyncer.go

```diff
@@ -330,6 +342,26 @@ func InitMetadataSyncer(ctx context.Context, clusterFlavor cnstypes.CnsClusterFl
 				return logger.LogNewErrorf(log, "failed to start informer on %q instances. Error: %v",
 					cnsoperatorv1alpha1.CnsStoragePolicyQuotaSingular, err)
 			}
+
+			go func() {
+				isStorageQuotaM2Enabled := metadataSyncer.coCommonInterface.IsFSSEnabled(ctx, common.StorageQuotaM2)
+				if isStorageQuotaM2Enabled {
+					// TODO: Wait for fullSync to update "zones" value on all CnsVolumeInfo CRs
+
+					// Start informer on AvailabilityZone CRs
+					err = startAvailabilityZoneCRInformer(ctx, k8sConfig)
```

Comment:
You'd need a vc connection inside this, so pass in `metadataSyncer` as well.

Reply:
Yes, there is no need to send complete metadataSyncer, we can just send vc instance.
I will add it as part of next PR where I will work on adding property collector for hosts of the cluster.